server/views/pages/book.njk

```diff
@@ -15,7 +15,7 @@
 {% block body %}
 
   <script type="application/ld+json">
-    {{ book | jsonLd('contentLd') | safe }}
+
```

Comment:
Is there something to be updated here (a TODO:?), or should we just remove the `script` block?